services/jwk.py

```diff
@@ -73,3 +89,23 @@ def wrapper(*args, **kwargs):
     wrapper.__doc__ = func.__doc__
     wrapper.__name__ = func.__name__
     return wrapper
+
+
+def has_role(*roles):
+    def decorator(func):
+        jwkservice = JWKService()
+
+        def wrapper(*args, **kwargs):
+            authorization_header = request.headers.get("Authorization")
+            if authorization_header is None:
+                pass
```

Comment:
`pass` is used as a placeholder, and doesn't actually do anything. Is there supposed to be some other functionality here? If there is but you're saving it for a future commit, add a todo comment saying what needs to be done. If there isn't going to be anything there, then remove the if statement